oca_dependencies.txt

```diff
@@ -4,3 +4,4 @@ ingadhoc-odoo-argentina https://github.com/ingadhoc/odoo-argentina.git
 # ingadhoc-argentina-sale https://github.com/ingadhoc/argentina-sale.git
 # ingadhoc-reporting-engine https://github.com/ingadhoc/reporting-engine.git
 oca-sale_workflow https://github.com/OCA/sale-workflow.git
+ingadhoc-patches https://github.com/ingadhoc/patches.git
```

Comment:
I think that a comment with a todo should be added here in order to take into account when we merge base_name_search_mproved to oca/server-tools